packages/react/src/components/Table/index.js

```diff
@@ -99,6 +108,7 @@ Table.propTypes = {
   isSortable: PropTypes.bool,
   manualSorting: PropTypes.bool,
   onChangeSort: PropTypes.func,
+  initialState: PropTypes.shape({}),
```

Comment:
Is the `initialState` shape always the same? 
If so, could we add a bit more depth?

Like:
```
initialState: PropTypes.shape({
  sortBy: PropTypes.arrayOf(PropTypes.shape({
    id: PropTypes.string.isRequired,
    desc: PropTypes.bool
   }] }
  }))
}),
```

Reply:
It can contain a bunch of other stuff, but I'll update it to support only the `sortBy` for now :+1: